cre/src/state_handler.cpp

```diff
@@ -0,0 +1,61 @@
+// Copyright (c) 2018-2021 VMware, Inc. All Rights Reserved.
+//
+// This product is licensed to you under the Apache 2.0 license (the "License"). You may not use this product except in
+// compliance with the Apache 2.0 License.
+//
+// This product may include a number of subcomponents with separate copyright notices and license terms. Your use of
+// these subcomponents is subject to the terms and conditions of the subcomponent's license, as noted in the LICENSE
+// file.
+
+#include "state_handler.hpp"
+#include "concord.cmf.hpp"
+#include <variant>
+namespace cre::state {
+bool sendUpdate(
```

Comment:
Why do we need directly add a block? I see the usage in `WedgeUpdateChainHandler` but shouldn't a block be constructed by a reconfiguration handler in the replica?

Reply:
Changed. 
We only send a write request with the update, we don't add blocks (and we can't we are in the client)

Reply:
fixed